gpMgmt/bin/gplogfilter

```diff
@@ -361,6 +361,12 @@ if (options.zip is None and
     options.zip = '9'
 
 try:
+    if (begin and end) and begin >= end:
+        raise IOError(
+            'Invalid arguments: "begin" date (%s) is >= "end" date (%s)'
```

Comment:
Why can't I request the same begin and end date?

Reply:
> As it [described](https://github.com/arenadata/gpdb/blob/69cdaf618dc2561c5da5ffcd7fdc61dcccfa5e3c/gpMgmt/bin/gppylib/logfilter.py#L84-L91) the beginstamp is inclusive and endstamp is exclusive (also it [described at TimestampInBounds function](https://github.com/arenadata/gpdb/blob/69cdaf618dc2561c5da5ffcd7fdc61dcccfa5e3c/gpMgmt/bin/gppylib/logfilter.py#L498-L501) which is [called from FilterLogEntries](https://github.com/arenadata/gpdb/blob/69cdaf618dc2561c5da5ffcd7fdc61dcccfa5e3c/gpMgmt/bin/gppylib/logfilter.py#L150-L151))